suite-native/module-accounts-import/src/components/AccountImportOverview.tsx

```diff
@@ -48,7 +35,7 @@ export const AccountImportOverview = ({ accountInfo, networkSymbol }: AssetsOver
                     <Box marginLeft="medium">
                         <Text>{networks[networkSymbol].name}</Text>
                         <Text variant="label" color="gray1000">
-                            {CryptoAmountFormatter.format(cryptoAmount, {
+                            {CryptoAmountFormatter.format(accountInfo.availableBalance, {
```

Comment:
Prefer component version of formatters.

Reply:
https://github.com/trezor/trezor-suite/pull/7401/commits/1cc34593cd42d928be9c517ad6ad4da0f00dc9a3